test_doubly_linkedlist.py

```diff
@@ -0,0 +1,135 @@
+import pytest
+
+from doubly_linkedlist import Dll
+
+
+def test_dll():
+    dll = Dll([4, 6, 8, 2])
+
+    assert dll.head.value == 2
+    assert dll.head.next.value == 8
+    assert dll.head.next.next.previous.value == 8
+    assert dll._length == 4
+    assert dll.tail.value == 4
+
+
+def test_push():
+    dll = Dll([5, 7, 8])
+    dll.push(3)
+
+    assert dll.head.value == 3
+    assert dll._length == 4
+    assert dll.head.previous == None
+    assert dll.head.next.value == 8
+    assert dll.head.next.previous.value == 3
+    assert dll.tail.value == 5
```

Comment:
tail.previous == None

Reply:
wouldn't tail.previous == the previous node and then tail.next == None?